packages/trader/src/sass/app/_common/components/contract-type-info.scss

```diff
@@ -12,52 +13,19 @@
         }
     }
 
-    & .dc-carousel {
```

Comment:
@henry-deriv why do we have style changes in (all scss files in this PR) if it's a tech debt PR? :) Is it not a tech debt PR?